app/tests/test_models.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+
+from core import models
+from tests import create_new_user, create_new_project, create_new_activity, finish_activity
+
+
+class ModelTests(TestCase):
+
+    def setUp(self):
+        self.user = create_new_user("user1")
+        self.project = create_new_project(user=self.user)
+
+    def test_project_str(self):
+        """Test the api string representation"""
+        project = create_new_project(user=self.user)
```

Comment:
why are u creating new project here even if you have one in setUp? why not reuse?

Reply:
OK, done